packages/neos-ui-editors/src/Editors/Button/style.module.css

```diff
@@ -0,0 +1,8 @@
+.buttonEditor {
+}
+
+.buttonEditor .button {
+    margin-right: var(--spacing-Quarter);
+    margin-top: var(--spacing-Quarter);
+    font-size: 20px;
+}
```

Comment:
```suggestion
.button.button {
    margin-right: var(--spacing-Quarter);
    margin-top: var(--spacing-Quarter);
    font-size: 20px;
}
```

No need for a base class. `.button.button` is for the desired specificity.

Also, I don't think the increased font-size is a good idea, because it's quite inconsistent with the rest of the UI:

|14px default|20px increased|
|-|-|
| ![image](https://github.com/neos/neos-ui/assets/2522299/ba61f3f6-e4db-446f-966d-a5a83b1882d7) | ![image](https://github.com/neos/neos-ui/assets/2522299/4a3d56e1-d1a6-4e0d-a17d-5e2fe675f6f0) |

Reply:
I've removed the font-size and added the "dirty state highlighting"